### ISSUE TITLE
Swiper组件仅有一张图片时候，在iOS 下 点击页面 变空白

### ISSUE THREAD

**Issue by litoxy** [airyland/vux#1560]
<swiper loop :aspect-ratio='ratio' dots-position="center" :show-desc-mask="showmask" :list="getImgs()"  v-if="goodsdetail.imgs.length>0" ></swiper>

getImgs定义如下:
```javascript
getImgs:function () {
        let bannerList = this.goodsdetail.imgs.map((one, index) => ({
          url: '#',
          img: one
        }))

        return bannerList;
      },

getDetail:function () {
          let _this = this;
        this.ajax.post('store/getimglist',{"goodsid":this.goodsid})
          .then(function (res) {
            _this.goodsdetail = res.data.data.pinfo;
          }).catch(function (err) {
            console.log(err)
          })
      }
```

如果 接口  store/getimglist 返回的 imgs 值有一条数据 如：
```
{
  "status": "200",
  "message": "\u6210\u529f",
  "data": {
    "pinfo": {
      "inventorynum": 99999,
      "state": 1,
      "piid": "107",
      "imgs": [
        "https:\/\/aimgstoreapi.moveclub.cn\/uploads\/image\/20170613\/593fa100c3970.jpg"
      ],
      "tips": "\u7acb\u5373\u8d2d\u4e70",
      "canpurchase": 1
    }
  }
}
```
则在IOS下 点击 页面 会变成空白
如果只有一张图片 会正常显示。但是 如果点击页面 整个页面会变成空白

**Comment by airyland** [comment]
issue 可以 reopen 的，不必要再建一个新的。

"点击页面"是什么意思

把业务代码删掉吧，直接给出我可以重现的代码。

**Comment by airyland** [comment]
按照你说的通过 setTimeout 模拟接口返回一张图片，并不能重现你所说的。麻烦补充 repo 或者  demo 页面再 reopen。
